python-package/lightgbm/engine.py

```diff
@@ -514,11 +510,11 @@ def _make_n_folds(
             train_set, valid_set, tparam = fpreproc(train_set, valid_set, params.copy())
         else:
             tparam = params
-        cvbooster = Booster(tparam, train_set)
+        booster_for_fold = Booster(tparam, train_set)
```

Comment:
I found it confusing at first that this was named `cvbooster` but was not actually a `CVBooster` object.

Thought it made sense in the context of this PR to change this variable name.